### DISCUSSION TITLE
Antialiasing

### DISCUSSION THREAD

**Discussion by hasantahir** [ahrm/sioyek#248] [General]
Is it possible to configure the `mupdf` antialiasing levels through sioyek?

**Comment by ahrm** [comment]
No. You can set sioyek's antialiasing to linear (instead of nearest neighbor) using the following config:
```
linear_filter 1
```

**Comment by hasantahir** [comment]
Thanks, here's a comparison with the PDF reader Skim (figures on the left):

Default Settings:
<img width="1918" alt="image" src="https://user-images.githubusercontent.com/11236148/170004006-5777e699-f11b-4d23-b104-9c65c4cedda0.png">


Linear Filter:
<img width="961" alt="image" src="https://user-images.githubusercontent.com/11236148/170004090-8eb509b2-b2e1-4f7b-9bed-62ca1afeb766.png">

Although the difference is subtle, but is there a way to achieve similar rendering? `Linear_filter` doesn't seem to have an effect. I put it in my `/.config/sioyek/prefs_user.config` file.

**Reply by ahrm** [reply]
I don't think it is possible.

**Reply by hasantahir** [reply]
Okay, thanks.
